post_sign.py

```diff
@@ -0,0 +1,48 @@
+# Koji callback for writing signed rpms after signing via sign.py
+# Required for package generation with strict keys in mash
+
+from koji.plugin import register_callback
+import kojihub
+import logging
+from configparser import ConfigParser
+
+# Configuration file in /etc like for other plugins
+config_file = '/etc/koji-hub/plugins/sign.conf'
+
+logger = logging.getLogger('koji.plugin.post_sign')
+
+
+def post_sign(cbType, sigkey=None, sighash=None, build=None, rpm=None):
+    """ Run the kojihub write-signed-rpm command on rpms after they've been signed with a signing key,
+    which is required for generating repos that use a specific key.
+    """
+    if sigkey is None or sighash is None or build is None or rpm is None:
+        logger.warning("Got unexpected None argument to post_sign plugin. Skipping")
```

Comment:
I know it makes the code longer but it would be useful to know _which_ argument was None.